nebula_common/include/nebula_common/continental/continental_srr520.hpp

```diff
@@ -0,0 +1,419 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <nebula_common/nebula_common.hpp>
+#include <nebula_common/nebula_status.hpp>
+
+#include "boost/endian/buffers.hpp"
+
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <bitset>
+#include <cmath>
+#include <cstddef>
+#include <cstdint>
+#include <ctime>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+namespace nebula
+{
+namespace drivers
+{
+namespace continental_srr520
+{
+
+/// @brief struct for SRR520 sensor configuration
+struct ContinentalSRR520SensorConfiguration : CANSensorConfigurationBase
+{
+  std::string base_frame{};
+  bool sync_use_bus_time{};
+  float configuration_vehicle_wheelbase{};
+};
+
+/// @brief Convert ContinentalSRR520SensorConfiguration to string (Overloading the <<
+/// operator)
+/// @param os
+/// @param arg
+/// @return stream
+inline std::ostream & operator<<(
+  std::ostream & os, ContinentalSRR520SensorConfiguration const & arg)
+{
+  os << (CANSensorConfigurationBase)(arg) << ", BaseFrame: " << arg.base_frame
+     << ", SyncUseBusTime: " << arg.sync_use_bus_time
+     << ", ConfigurationVehicleWheelbase: " << arg.configuration_vehicle_wheelbase;
+  return os;
+}
+
+/// @brief semantic struct of ARS548 Status
```

Comment:
SRR520

Reply:
Addressed in [54b900f](https://github.com/tier4/nebula/pull/124/commits/54b900f2d3c8121868ea3a60bc7609732b110701)
:bow: